### ISSUE TITLE
Cut a release?

### ISSUE THREAD

**Issue by felixonmars** [fossasia/susi_api_wrapper#64]
It would be nice to have a versioned release which an initial PyPI upload and distribution packaging could start with.

**Comment by felixonmars** [comment]
I understand that development is still ongoing and could change very often - but releases can really help start all the packaging bits. If you have trouble with PyPI I can help with all the uploading & updating etc given there are git tags.

The project status field in setup.py could always be used to indicate that it's still in early stage, just in case you have concern in releasing unstable things.

I would like to kick start the packaging bits in this year's GSoC, and this is one of the bottom-level dependencies of susi_linux tree.

**Comment by mariobehling** [comment]
Please go ahead with packaging. Would be great!

**Comment by hongquan** [comment]
Honestly, I'm not consent about the design of this piece of software (susi_api_wrapper). I think it is better to reconstruct it before doing packaging.

I'm sorry that I don't have time to do it yet.

**Comment by cseas** [comment]
Can I work on creating a PyPi package for susi_python?

**Comment by norbusan** [comment]
@hongquan I agree with you that the design/layout is not optimal, in particular because the current repository combines in principle api wrappers for multiple languages.

I would suggest the following;
- rename the repository from `susi_api_wrapper` to `susi_python`
- move the files from `susi_api_wrapper/python_wrapper` one level up, so directly into the `susi_python` repository
- if need arrives, create new repos for different languages, like `susi_mono` etc ..
WDYT?

@cseas What do you think is needed there? A simple
```
cd susi_api_wrapper.git/python_wrapper
python3 setup.py bdist_wheel
```
already produces .whl packages. What else are you planning to do?

**Comment by hongquan** [comment]
@norbusan Agree!

**Comment by norbusan** [comment]
> @norbusan Agree!

Can you create a `susi_python` repo on github? Then I will move the current files over to preserve the history, and restructure it.

**Comment by cseas** [comment]
@norbusan I wasn't aware of the wheel generation. The README doesn't mention any wheel generation so I thought we just download the repository and run the program inside it by importing `susi_python`.

I was thinking of releasing a PyPi package for this so the module can be install with `pip install susi_python` from any terminal. That would make it easier for people to start using and experimenting with it.

**Comment by norbusan** [comment]
@cseas Yes, that is for sure the idea. I haven't seen any releases so we could very well create a 0.0.1 release, make a whl package, and upload it to the fossasia pip repository.

**Comment by norbusan** [comment]
@hongquan we could also physically rename the repository, but this is a bit brutal ... and would need adaption of the current raspi image generation (not mine, but the one that generates the current images by @Orbiter)

**Comment by cseas** [comment]
@norbusan That sounds good. Should I look into creating the PyPi package and see if it works well?

**Comment by norbusan** [comment]
@cseas that is for sure a good idea. Things that need to be checked is whether e.g the RPi image with the normal susi_python removed from SUSI.AI folder, but installed via pip, then the susi_linux app still is working.

**Comment by cseas** [comment]
@norbusan Okay, I'll check it out.

**Comment by hongquan** [comment]
@norbusan Added the repo `susi_python`.

**Comment by norbusan** [comment]
@hongquan I have moved the files with history, and added also Debian packaging. We should probably remove this repository or archive it?

**Comment by hongquan** [comment]
@norbusan Archive it.
